src/test/java/org/opentripplanner/test/support/PrettyPrinter.java

```diff
@@ -0,0 +1,39 @@
+package org.opentripplanner.test.support;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.util.DefaultIndenter;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+public class PrettyPrinter {
```

Comment:
Why not use `org.opentripplanner.standalone.config.framework.JsonSupport`?

Reply:
Do you mean I should move the pretty-printing functionality there?

Reply:
I've moved the pretty-printing into that class and moved the class into the correct package.